### DISCUSSION TITLE
Game Setup option: Construction Queue Spillover Yes/No

### DISCUSSION THREAD

**Discussion by Pelldom** [ekolis/FrEee#236] [Ideas]
Just a thought, but what if the way construction queues "spilled over" construction to the next item in the queue (like SEV) could be turned on or off as a game setup option?

So you could have it ON (SEV spillover)
or OFF (SEIV no spillover)

**Comment by Pelldom** [comment]
Or, if not a game setup option, what about a Settings.txt option?

**Comment by ekolis** [comment]
That sounds like a great idea! If I'm not mistaken, FrEee goes even beyond what SE5 will do with spillover, allowing two items to build simultaneously if they require different resources, even without the first one being completed. So we could actually have three options for spillover - SE4 style no spillover, SE5 style spillover, and FrEee style insane spillover!

As for game setup vs. settings.txt, a lot of the settings.txt options could honestly be converted to game setup options, but then I'd have to make a UI for editing them...

**Reply by Pelldom** [reply]
UI project!

I better get cracking on my lessons...
